testing/simapp/app.go

```diff
@@ -768,6 +832,16 @@ func (app *SimApp) InterfaceRegistry() types.InterfaceRegistry {
 	return app.interfaceRegistry
 }
 
+// TxConfig returns SimApp's TxConfig
+func (app *SimApp) TxConfig() client.TxConfig {
+	return app.txConfig
+}
+
+// DefaultGenesis returns a default genesis from the registered AppModuleBasic's.
```

Comment:
grammer nit, reworded a suggestion

```suggestion
// DefaultGenesis returns a raw json message containing the AppModuleBasic's default genesis state.
```